### PR TITLE
add homeassistant

### PR DESCRIPTION
**Pull request by tprelog**
[Home Assistant](https://www.home-assistant.io/)

Requested here: https://redmine.ixsystems.com/issues/54990

### CODE REVIEW

File: homeassistant.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "homeassistant",
+  "release": "11.3-RELEASE",
+  "artifact": "https://github.com/tprelog/iocage-homeassistant.git",
+  "official": "false",
+  "properties": {
+    "allow_raw_sockets": "1",
+    "boot": "on",
```
Comment (miwi-fbsd) [inline homeassistant.json]: please keep this optional

File: homeassistant.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "bpf": "on",
```
Comment (miwi-fbsd) [inline homeassistant.json]: Please remove it automated now.

File: homeassistant.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "dhcp": "1",
+    "vnet": "yes"
```
Comment (miwi-fbsd) [inline homeassistant.json]: Please remove it automated now.

**Comment by miwi-fbsd** [comment]
Hi,

Getting,

  - py-sqlite3...
    - py-sqlite3 failed to install, retry #1
    - py-sqlite3 failed to install, retry #2
    - py-sqlite3 failed to install, retry #3
  - libxml2...
    - libxml2 failed to install, retry #1
    - libxml2 failed to install, retry #2
    - libxml2 failed to install, retry #3
  - libxslt...
    - libxslt failed to install, retry #1
    - libxslt failed to install, retry #2
    - libxslt failed to install, retry #3
  - sudo...
    - sudo failed to install, retry #1
    - sudo failed to install, retry #2
    - sudo failed to install, retry #3
  - wget...
    - wget failed to install, retry #1
    - wget failed to install, retry #2
    - wget failed to install, retry #3

Could you please check?

**Comment by tprelog** [comment]
Sorry pkg name was different without the category. It's fixed now

**Comment by miwi-fbsd** [comment]
Hi,
I've open 2 pull request to fix the permission,
https://github.com/tprelog/iocage-homeassistant/pull/3
https://github.com/tprelog/iocage-homeassistant/pull/4

When that is fixed I'll make one more round of testing but it looks good to go :)

**Comment by tprelog** [comment]
Fixed! Finger crossed

**Pull request merged**